spec/jobs/trainees/set_cohort_job_spec.rb

```diff
@@ -6,9 +6,13 @@ module Trainees
   describe SetCohortJob do
     let(:trainee) { double }
 
-    it "calls SetCohort with the supplied trainee" do
-      expect(SetCohort).to receive(:call).with(trainee: trainee)
-      described_class.perform_now(trainee)
+    context "feature enabled" do
+      before { enable_features(:set_trainee_cohort) }
+
+      it "calls SetCohort with the supplied trainee" do
+        expect(SetCohort).to receive(:call).with(trainee: trainee)
+        described_class.perform_now(trainee)
+      end
```

Comment:
Is it worth adding a test for the feature flag off case?

Reply:
Yep. Done 👍